test/e2e/atlas/dbusers_test.go

```diff
@@ -156,8 +159,25 @@ func TestDBUsersWithStdin(t *testing.T) {
 		testCreateUserCmd(t, cmd, username)
 	})
 
+	t.Run("Create OIDC user", func(t *testing.T) {
+		require.NoError(t, err)
```

Comment:
seems like the wrong place for this line, as where is this error coming from?

Reply:
yeah i. don't know how this got there, I'll fix!